### PR TITLE
Update setuptools to address package_index vulnerability

### LINKED ISSUE #50: Address security issue in setuptools < 70.0
https://github.com/advisories/GHSA-cx63-2mw6-8hw5

### PR DESCRIPTION
**Pull request by northwestwitch**
## Description
- Closes #50 

### Fixed
- Docker image should build without errors and pushed to Docker Hub (doesn't work unless we update setuptools in [loqusdb repo first](https://github.com/Clinical-Genomics/loqusdb/pull/133))

### How to test
- [x] Make sure the Docker image builds and gets pushed to Docker Hub

## Review

- [x] Tests executed by GitHub actions
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.1.14]
+### Fixed
+- Update to setuptools >= v.70 to address a security issue in the `package_index` module
+
+## [0.1.13]
```
Comment (northwestwitch) [inline CHANGELOG.md:9]: This was updated during last release

**Comment by northwestwitch** [comment]
Docker build and push to [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/loqusdbapi-stage/general) went fine ->

image

**Review by diitaz93 — APPROVED** [review]
👍

**Review by ivadym — APPROVED** [review]
Nice 💯

**Pull request merged**